src/Data/JSDate.purs

```diff
@@ -150,8 +150,11 @@ foreign import dateMethod :: forall a. Fn2 String JSDate a
 
 -- | Attempts to parse a date from a string. The behaviour of this function is
 -- | implementation specific until ES5, so may not always have the same
--- | behaviour for a given string. The RFC2822 and ISO8601 date string formats
--- | should parse consistently.
+-- | behavior for a given string. For this reason, it is **strongly** encouraged
+-- | that you avoid this function if at all possible.
```

Comment:
We could add "and instead use functions from a higher-level library like `datetime`" but I'll defer to @hdgarrood on his preferred wording.

Reply:
This seems fine to me :+1: